src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void addList(String line) {
+        System.out.println("--------------------------------------");
+        System.out.println("added: " + line);
+        System.out.println("--------------------------------------");
+    }
+
+    public static void printBye() {
+        System.out.println("--------------------------------------");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("--------------------------------------");
+    }
+
+    public static void printList(Task[] list, int listIndex) {
+        int printCounter = 1;
+        System.out.println("--------------------------------------");
+        System.out.println("Here are the tasks in your lists:");
+        for (Task item : Arrays.copyOf(list, listIndex)) {
+            System.out.println(printCounter + "." + item.getStatusIcon() + " " + item.description);
+            printCounter++;
+        }
+    }
+
+    public static void introStart() {
+        String logo
+                = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("--------------------------------------");
+        System.out.println("Hello! I'm Steffy");
+        System.out.println("What can I do for you?");
+        System.out.println("--------------------------------------");
+    }
+
+    public static void main(String[] args) {
+
+        introStart(); //Prints starting screen
+
+        boolean ifExit = false; //exits program if true
```

Comment:
I think you can consider renaming the boolean variable since the word "ifExit" might not sound like a boolean variable name.